deploy/templates/nstemplatetiers/advanced/based_on_tier.yaml

```diff
@@ -0,0 +1,9 @@
+from:
+  name: base
+to:
+  name: advanced
+  parameters:
```

Comment:
Should we put the parameters field at the top level in case we want to expand on the composition in the future?
eg.
```
from:
   name: base
with:
   name: advanced-networkpolicies
to:
   name: advanced
parameters:
  - name: DEACTIVATION_TIMEOUT_DAYS
    value: 0
  - name: IDLER_TIMEOUT_SECONDS
    value: 0
  - name: NETWORK_PARAM
    value: 10
```

Reply:
Or maybe it makes more sense for the parameters to go with the source template?
eg.
```
from:
   name: base
   parameters:
     - name: DEACTIVATION_TIMEOUT_DAYS
        value: 0
     - name: IDLER_TIMEOUT_SECONDS
        value: 0
with:
   name: advanced-networkpolicies
   parameters:
    - name: NETWORK_PARAM
       value: 10
to:
   name: advanced
```

Reply:
Or I'm fine with keeping it as is and dealing with it if/when we actually have a need. 🙂

Reply:
I used this structure because it seemed logical to me to put all new parameters into `to:` section. In other words, it defines what is the new tier and which new parameters/values are used there.
But I'm open to discussion, let's discuss it in our sync call. I'll merge the PRs to unblock the other work on tiers. Change of the structure should be pretty simple.